.github/ISSUE_TEMPLATE/registration-issues.md

```diff
@@ -25,6 +25,10 @@ If all the above looks good you can open an issue.
 
 **Link to the intermediate certificate on the [EU Trusted list](https://eidas.ec.europa.eu/efda/tl-browser/#/screen/home):**
 
+**Debug information**
+
+* Call `https://debug.openbanking-sandbox.prod.lunar.tech` including your certificate and private key (eg. `curl -k https://debug.openbanking-sandbox.prod.lunar.tech --cert your_certificate --key your_private_key`)
```

Comment:
* Any reason we use `-k`? Are the TLS certs self-signed?
* Consider using an extension for the files
* Consider postfixing an `s` to `your_certificate` to hint that the file (might) contain multiple certs 

```suggestion
* Call `https://debug.openbanking-sandbox.prod.lunar.tech` including your certificate and private key (eg. `curl https://debug.openbanking-sandbox.prod.lunar.tech --cert your_certificates.pem --key your_private_key.pem`)
```

Reply:
* The certificate is registered to `tpp.openbanking-sandbox.prod.lunar.tech`... I guess we should fix that
* I intentionally did not use an extension as I do not know that it needs to be in a specific format. Does it only accept PEM-format?
* Good point

Reply:
I looked up the `curl` documentation: `--key` assumes it is PEM, because `--key-type` is not specified https://curl.se/docs/manpage.html#--key-type. 

Would it make sense to put `.pem` to `your_private_key`?